modules/proxy.js

```diff
@@ -48,6 +98,14 @@ module.exports = async ({msg,bot,members,cfg}) => {
 					process.send({name: "queueDelete", channelID: msg.channel.id, messageID: msg.id}, null, {swallowErrors: false}, err => {
 						if(err) console.log(err)
 					});
+				//if I had set an auto proxy with -tul!auto then write that to the database
+				if(setAutoProxy){
+					client = await bot.db.connect();
```

Comment:
Please don't launch DB connections in files. Instead, just call the bot.db.method, which will use `bot.db`'s connection.

Reply:
removed - used the pool instead in db.js